python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/stubbed_ecs.py

```diff
@@ -150,14 +152,42 @@ def describe_tasks(self, **kwargs):
         )
         return self.client.describe_tasks(**kwargs)
 
+    @stubbed
+    def list_account_settings(self, **kwargs):
+        """
+        Only taskLongArnFormat has a default value
+        """
+        if kwargs.get("effectiveSettings"):
+            account_settings = {
+                **self.default_account_settings,
+                **self.account_settings,
+            }
+        else:
+            account_settings = self.account_settings
+
+        account_settings = [
+            {
+                "name": key,
+                "value": value,
+            }
+            for key, value in account_settings.items()
+        ]
+
+        self.stubber.add_response(
+            method="list_account_settings",
+            service_response={"settings": account_settings},
+            expected_params={**kwargs},
```

Comment:
what's the diff between {**kwargs} and kwargs? shallow copy?

Reply:
Just cargo-culting what I had done with the other stubs - I don't remember if I had a specific reason for choosing one over the other.